source/extensions/filters/http/jwt_authn/authenticator.h

```diff
@@ -25,24 +26,16 @@ class Authenticator {
 
   // Verify if headers satisfyies the JWT requirements. Can be limited to single provider with
   // extract_param.
-  virtual void
-  verify(Http::HeaderMap& headers, std::vector<JwtLocationConstPtr>&& tokens,
-         std::function<void(const ::google::jwt_verify::Status& status)>&& callback) PURE;
+  virtual void verify(Http::HeaderMap& headers, std::vector<JwtLocationConstPtr>&& tokens,
+                      AuthenticatorCallback&& callback) PURE;
```

Comment:
for callback, not need to use &&.  std::function is the same as shared_ptr.   not ownership transfer.